packages/orchestration/src/proposals/orchestration-proposal.js

```diff
@@ -86,6 +108,12 @@ export const getManifestForOrchestration = (_powers, { orchestrationRef }) => ({
         orchestrationVat: 'orchestrationVat',
       },
     },
+    [initChainInfo.name]: {
+      consume: {
+        agoricNames: true,
```

Comment:
it's harmless, but why is `agoricNames` here?

Reply:
thanks. leftover from an earlier state. I'll remove